website/docs/feature-flag-tutorials/react/implementing-feature-flags.md

```diff
@@ -5,29 +5,22 @@ slug: /feature-flag-tutorials/react
+[Cypress Real World App](https://github.com/cypress-io/cypress-realworld-app) is an open-source React project that allows you to test and experiment in a React codebase that mirrors a real-world use case: a financial transaction application. It harnesses Cypress for testing, Material UI for CSS, a built-in database, and other tools to provide a fully functioning application experience for educational purposes.
 
 ![Cypress Real World App](/img/react-tutorial-rwa.png)
 
-In this tutorial, you will learn how to set up and use React feature flags with Unleash. We will be using the Cypress Real World App to implement the feature flag solution, which will simulate how you can gradually roll out a notifications feature to users. At the end of this tutorial, you'll be able to enable on the flag you create and launch the notification icon, making it visible for a percentage of users to click on and be taken to a Notifications view to see transaction updates from user contacts.
+In this tutorial, you will learn how to set up and use React feature flags with Unleash. We will be using the Cypress Real World App to implement the feature flag solution, which will simulate how you can gradually roll out a notifications feature to users. By the end of this tutorial, you will be able to activate your created flag and unveil the notification icon to a select group of users, who can then click on it to access the Notifications view and view transaction updates from their contacts.
 
 Along the way, you will:
 
-1. [Spin up a local instance of Unleash](#1-install-and-run-unleash-on-your-local-machine)
-2. [Create a feature flag](#2-create-and-enable-a-feature-flag)
-3. [Generate an API token](#3-generate-an-api-token)
-4. [Clone a React app](#4-clone-an-open-source-react-app)
-5. [Set up Unleash in your app](#5-set-up-unleash-in-your-app)
-6. [Toggle the visibility of a feature component](#6-use-the-feature-flag-to-rollout-a-notifications-badge)
-7. [Verify the toggle experience](#7-verify-the-toggle-experience)
-
-
-## Considerations for using feature flags with React
-
-
-We recommend that you reduce sensitive user data exposure by conducting feature flag evaluations in a self-hosted environment. Evaluating on the client side in a React application could potentially **[expose sensitive data](https://docs.getunleash.io/topics/feature-flags/never-expose-pii)**  such as API keys, flag configurations and flag data. A server-side evaluation of feature flags is recommended practice for privacy protection, as it will minimize sending data to the Feature Flag Control Service and reduce the attack surface of your application/services.
+1. [Architect to limit PII and configuration leakage](#1-architect-to-limit-pii-and-configuration-leakage)
+2. [Spin up a local flag provider](#2-install-a-local-feature-flag-provider)
+3. [Configure a feature flag](#3-create-enable-and-configure-a-feature-flag​)
```

Comment:
For some reason this link (#3) is broken in the build. I don't see an issue in the web editor, but maybe a line ending or blank space issue? 
Please troubleshoot and fix before we push.